builtin/audit/file/backend.go

```diff
@@ -60,6 +62,21 @@ func Factory(_ context.Context, conf *audit.BackendConfig, useEventLogger bool,
 		return nil, fmt.Errorf("%s: nil salt view", op)
 	}
 
+	// The config options 'fallback' and 'filter' are mutually exclusive, a fallback
```

Comment:
This same chunk of code and its tests are repeated for each of the audit backends. Can this instead be moved to a shared function?